lib/importer.js

```diff
@@ -113,7 +113,10 @@ var importChangesBatch = function(db, couchdb, concurrentDocLimit, changesLimit,
       log.debug('Downloading CouchDB changes feed from ' + seq);
       return couchdb.changes({
         limit: changesLimit,
-        since: seq
+        since: seq,
+        filter: function (doc) {
+          return !doc.views;
```

Comment:
We probably don't want to do this in `couch2pg`.

It's a fine idea to restrict what we download, but we'd want to do it externally to couch2pg, presumably as something passed from medic-couch2pg.

This is a generic tool. Just because we don't want to pull down ddocs doesn't mean someone else does not.

Also, if you want to check for a ddoc you'd want to check that the _id starts with `_design/`, as any document can have a views property.